src/OSPSuite.Core/Serialization/Exchange/SimulationTransferXmlSerializer.cs

```diff
@@ -12,8 +12,12 @@ public override void PerformMapping()
          Map(x => x.Simulation);
          Map(x => x.JournalPath);
          Map(x => x.Favorites);
-         MapEnumerable(x => x.AllObservedData, x => x.AllObservedData.Add);
          Map(x => x.PkmlVersion).WithMappingName(Constants.Serialization.Attribute.VERSION);
+
+         MapEnumerable(x => x.AllObservedData, x => x.AllObservedData.Add);
+         //This needs to be done after observed data serialization to ensure that the observed data were deserialized
+         Map(x => x.OutputMappings);
```

Comment:
One little trick here. It needs to be done after observed data are deserialized otherwise the ref to obs data won't be found